generator/src/templates/assembly-package.mst

```diff
@@ -18,16 +18,13 @@
     "@theia/console": "^{{ version }}",
     "@theia/core": "^{{ version }}",
     "@theia/debug": "^{{ version }}",
-    "@theia/debug-nodejs": "^{{ version }}",
```

Comment:
just a note: should debug nodejs VS Code extension should be added somehow in the typescript che plugin ?

Reply:
@benoitf I think yes, tslint and nodejs VS Code extension should be included into typescript che plugin
I'll propose a PR to plugin registry

Reply:
provided PR https://github.com/eclipse/che-plugin-registry/pull/400